assets/javascripts/obs_rsync.js

```diff
@@ -1,22 +1,19 @@
 function handleObsRsyncAjaxError(xhr, ajaxOptions, thrownError) {
-  var message = xhr.responseJSON.error;
-  if (!message) {
-    message = thrownError ? thrownError : 'no message';
-  }
-  addFlash('danger', 'Error: ' + message);
+  const message = xhr.responseJSON?.error ?? thrownError ?? 'no message';
+  addFlash('danger', `Error: ${message}`);
 }
 
 function fetchValue(url, element, controlToShow) {
   $.ajax({
-    url: url,
+    url,
```

Comment:
Even if it works without `url: ` I think it is better to use the key explicitly. That makes the code much easier to read for people which are not that familiar with JavaScript.

Reply:
Yes, we can use the shorthand syntax for object property definition in JavaScript to write `url` instead of `url: url`

Reply:
Using the shorthand `url` instead of `url: url` is considered good JavaScript style now.

Reply:
It goes hand in hand with how we now use destructuring for assignments, like `const {host} = url;` instead of `const host = url.host;`.